### PR TITLE
Move to single family font-face declaration + fix faux bold and faux italic

### PR DESCRIPTION
**Pull request by ryanditjia**
Here is the PR as we talked about on Twitter. Apparently it wasn’t only faux bold, some italics (e.g. blog post) are also faux italic, always thought it looked too slanted 😄.

Happy to answer any questions you might have.

### CODE REVIEW

File: src/components/confirm-message/illustrations.js [hunk 1]
```diff
@@ -146,9 +146,9 @@ export const PleaseConfirmIllustration = (
         />
         <text
           fill="#FFF"
-          fontFamily={fonts.bold}
+          fontWeight={fontWeights.bold}
           fontSize="10"
-          fontWeight="600"
+          // fontWeight="600" // need clarification
```
Comment (ryanditjia) [inline src/components/confirm-message/illustrations.js:151]: I’m not sure which weight you want, so I went with the one you set with fontFamily

File: src/components/confirm-message/illustrations.js [hunk 2]
```diff
@@ -371,9 +371,9 @@ export const ThankYouIllustration = (
           />
           <text
             fill="#FFF"
-            fontFamily={fonts.regular}
+            fontWeight={fontWeights.regular}
             fontSize="14.3"
-            fontWeight="500"
+            // fontWeight="500" // need clarification
```
Comment (ryanditjia) [inline src/components/confirm-message/illustrations.js:376]: I’m not sure which weight you want, so I went with the one you set with fontFamily

File: src/components/confirm-message/message.js [hunk 3]
```diff
@@ -72,16 +73,13 @@ export const Center = styled.div`
   p {
     margin-top: 10px;
     max-width: 400px;
+    font-weight: ${fontWeights.regular};
     line-height: 1.5;
-    font-weight: 400;
-    strong {
-      font-weight: 600;
```
Comment (ryanditjia) [inline src/components/confirm-message/message.js:78]: This strong will be set to 600 implicitly. Due to our global:

```css
b,
strong {
  font-weight: bolder;
}
```

File: src/components/layout.js [hunk 4]
```diff
@@ -35,23 +35,26 @@ export const globalStyles = css`
...
     background: none;
   }
-  em {
-    font-family: ${fonts.regularItalic};
-  }
   strong,
   b {
-    font-family: ${fonts.semibold};
-    em {
-      font-family: ${fonts.semiboldItalic};
```
Comment (ryanditjia) [inline src/components/layout.js:45]: By default, `em` has `font-style: italic`

File: src/components/mdx/subtitle.js [hunk 5]
```diff
@@ -1,10 +1,10 @@
 import styled from '@emotion/styled'
-import {rhythm} from '../../lib/typography'
+import {rhythm, fontWeights} from '../../lib/typography'
 
 export default styled.h2`
   text-align: left;
   margin-top: ${rhythm(2)};
   margin-bottom: ${rhythm(1)};
-  font-weight: 900;
+  font-weight: ${fontWeights.black};
```
Comment (ryanditjia) [inline src/components/mdx/subtitle.js:8]: This is the only black weight you’re using in the entire site. I feel the intended weight is bold, please check the result though.

File: src/lib/reset.js [hunk 6]
```diff
@@ -19,8 +19,6 @@ const reset = css`
   }
   html,
   body {
-    font-family: ${fonts.regular}, sans-serif;
```
Comment (ryanditjia) [inline src/lib/reset.js:22]: Redundant. Already set with Typography.

File: src/lib/typography.js [hunk 7]
```diff
@@ -1,41 +1,54 @@
...
+]
+
 const typography = new Typography({
   baseFontSize: '18px',
   baseLineHeight: 1.55,
   headerLineHeight: 1.4,
-  headerFontFamily: [fonts.light, 'sans-serif'],
-  bodyFontFamily: [fonts.regular, 'sans-serif'],
+  bodyFontFamily: ['Inter', ...systemFontStack],
+  headerFontFamily: ['Inter', ...systemFontStack],
```
Comment (ryanditjia) [inline src/lib/typography.js:36]: IMO looks better than just sans-serif. But happy to revert if you wish.

Reference: https://github.com/csstools/sanitize.css/blob/b1555b1445786588f99dda84ab8220345ec0d831/typography.css#L6-L18

File: src/fonts/fonts.css [hunk 8]
```diff
@@ -1,145 +1,146 @@
-@font-face {
```
Comment (ryanditjia) [inline src/fonts/fonts.css:1]: Commented out the unused weights:
Thin 100
ExtraLight 200
Medium 500
ExtraBold 800

File: src/lib/typography.js [hunk 9]
```diff
@@ -1,41 +1,54 @@
...
-  // lightItalic: 'Inter Light Italic',
-  regular: 'Inter Regular',
-  // regularItalic: 'Inter Regular Italic',
-  // medium: 'Inter Medium',
-  // mediumItalic: 'Inter Medium Italic',
-  semibold: 'Inter Semibold',
-  // semiboldItalic: 'Inter Semibold Italic',
-  bold: 'Inter Bold',
-  // boldItalic: 'Inter Bold Italic',
+export const fontWeights = {
```
Comment (ryanditjia) [inline src/lib/typography.js:4]: Commented out the unused weights:
Thin 100
ExtraLight 200
Medium 500
ExtraBold 800

**Comment by kentcdodds** [comment]
Hey @ryanditjia,

Thank you for the PR. I'm not sure I like the changes. I like the way the fonts look today on the site.

**Pull request closed without merging**